README.md

```diff
@@ -2,7 +2,7 @@
 
 Kubernetes operator to deploy and manage [RabbitMQ](https://www.rabbitmq.com/) clusters. This repository contains a [custom controller](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/#custom-controllers) and [custom resource definition (CRD)](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/#customresourcedefinitions) designed for the lifecycle (creation, upgrade, graceful shutdown) of a RabbitMQ cluster.
 
-**Note**: this repository is under active development and is provided as **beta** software. Official support for this software is not provided; if you encounter any issues running this software, please feel free to [contribute to the project](#contributing).
+**Note**: this repository is under active development. Official support for this software is not provided; if you encounter any issues running this software, please feel free to [contribute to the project](#contributing).
```

Comment:
Should we state that official support can be purchased through VMware Tanzu?

Reply:
@yaronp68 do you think that's good idea?

Reply:
I'm ok to not mention the commercial support in the Readme, same as other RabbitMQ repos. We can direct people to the RabbitMQ web site for that: https://www.rabbitmq.com/#support